auto_submit/test/src/service/fake_config.dart

```diff
@@ -2,9 +2,26 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
+
 import 'package:auto_submit/service/config.dart';
+import 'package:auto_submit/service/github_service.dart';
+import 'package:github/github.dart';
 import 'package:neat_cache/neat_cache.dart';
 
+// Represents a fake config to be used in unit test.
 class FakeConfig extends Config {
-  FakeConfig() : super(cacheProvider: Cache.inMemoryCacheProvider(4));
+  FakeConfig({
+    this.githubClient,
+    this.githubService,
+  }) : super(cacheProvider: Cache.inMemoryCacheProvider(4));
+
+  Future<GitHub>? githubClient;
+  Future<GithubService>? githubService;
+
+  @override
+  Future<GitHub> createGithubClient() => githubClient!;
```

Comment:
Add `async` here should help you directly assign `githubService` in webhook test.

Reply:
Thank you Keyong! I will review the async part in Dart doc to have a more comprehensive study of this part of knowledge!